### PR TITLE
remove real IPs with reserved test/documentation IPs (rfc5737)

### PR DESCRIPTION
**Pull request by JakeDEvans**
#### What type of PR is this?
/kind cleanup
/kind documentation

#### What this PR does / why we need it:
Removes real public IPs with reserved ones (see RFC 5737)

#### Does this PR introduce a user-facing change?
NONE

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
@JakeDEvans can you please explain why we need this change. What problem does it solve?

File: pkg/kubelet/kubelet_pods_test.go [hunk 1]
```diff
@@ -185,17 +185,17 @@ fe00::0	ip6-localnet
...
 # Entries added by HostAliases.
-123.45.67.89	foo	bar	baz
+192.0.2.89	foo	bar	baz
 `,
 		},
 		{
 			"hosts_test_file2_with_host_aliases",
 			[]v1.HostAlias{
-				{IP: "123.45.67.89", Hostnames: []string{"foo", "bar", "baz"}},
-				{IP: "456.78.90.123", Hostnames: []string{"park", "doo", "boo"}},
```
Comment (aojea) [inline pkg/kubelet/kubelet_pods_test.go:198]: just curious, don't we validate ips? this IP is invalid

Comment (JakeDEvans) [reply pkg/kubelet/kubelet_pods_test.go:198]: yeah I noticed that too, apparently not

**Comment by JakeDEvans** [comment]
> @JakeDEvans can you please explain why we need this change. What problem does it solve?

using any real IPs in testing can have the unforeseen event of exposure either connecting to actual systems that use those IPs, data exposure or abuse.

**Comment by aojea** [comment]
> using any real IPs in testing can have the unforeseen event of exposure either connecting to actual systems that use those IPs, data exposure or abuse.

these are unit tests, there is no networking exercising here

**Comment by SergeyKanzhelev** [comment]
/close

Thank you for your PR. It seems to only contain trivial edits. Please read our [trivial edits policy](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits). We encourage you to take a look at confirmed issues and bugs or issues marked `help-wanted`.

**Pull request closed without merging**